tutorials/misc/jitter_stutter.rst

```diff
@@ -49,6 +49,16 @@ framerate performance. Solving this is game specific and will require optimizati
 The second is more complicated, because it is often not associated to the engine or game but the underlying operating system.
 Here is some information regarding stutter on different OSs.
 
+VSync
+^^^^^
+
+A common cause of Jitter is when the game's framerate is out of step with the monitor's refresh rate.  Go to ``` Project Settings -->  Display --> Window``` and ensure that ```Vsync``` is enabled, and if you are on Windows, also ensure that ```Vsync Via Compositor``` is enabled.
```

Comment:
It's `Use Vsync` not just `Vsync` also can you clarify where use needs to select `Vsync Via Compositor`  I don't know where this setting is. Finally `if you are using Windows` not if `if you are on Windows`